geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
 # Date first added: 2023-05-15 (by Nike Dattani)
 
 H2 Ar 3.957
+
+NAME = AgClH3N  2010
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Mikhailov VA, Tew DP, Walker NR, Legon AC (2010) H3N⋅⋅⋅Ag–Cl: Synthesis in a supersonic jet and
+characterization by rotational spectroscopy. Chem Phys Lett 499(1-3):16-20
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   Cl     2.26333(6)
+Ag...N      2.15444(6)
+N    H      1.0129
+H    N...Ag 113.48(2)
+
+# Bond lengths: rs     (angstroms)
+
+Ag   Cl     2.2633(9)
+Ag...N      2.1545(8)
+
+NAME = AgH3IN
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Medcraft C, Gougoula E, Bittner DM, Mullaney JC, Blanco S, Tew DP, Walker NR, Legon AC (2017)
+Molecular geometries and other properties of H2O⋅⋅⋅AgI and H3N⋅⋅⋅AgI as characterized by rotational
+spectroscopy and ab initio calculations. J Chem Phys 147(23):234308/1-234308/8
+https://doi.org/10.1063/1.5008744
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   I   2.5375(3)
+Ag...N   2.180(1)
```

Comment:
Looking back at what I did for the triatomics that I added earlier, I had:
```
H2 Ar 3.957
``` 
To denote the length between the center-of-mas of H2 and the Ar atom.
Similarly, something here would ideally denote that Ag is not just interacting with N, but with NH3.

Reply:
See above fix.